.github/workflows/add-issues-to-project.yml

```diff
@@ -4,8 +4,12 @@ on:
     types:
       - opened
 
```

Comment:
If you add
```push:```
to the `on:` block you can test these workflows before the PR is merged.

Reply:
https://github.com/ministryofjustice/data-platform/actions/runs/4083720472/jobs/7039578006
ive had to generate a PAT and give it the project scope because I cant change the GITHUB_TOKEN as the project is not at the repo level. This action run although failed proves that it has relevant permissions. It fails because on push it doesnt have the required data thats part of an issue event